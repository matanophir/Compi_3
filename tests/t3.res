---begin global scope---
print (string) -> void
printi (int) -> void
main () -> void
  ---begin scope---
  arr1 int 0
  x int 1
  ---end scope---
---end global scope---
