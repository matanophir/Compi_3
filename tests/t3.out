---begin global scope---
print (string) -> void
printi (int) -> void
main () -> void
  ---begin scope---
  x int 0
  y[17] int 1
  z byte 18
  w[3] bool 19
  a int 22
  ---end scope---
---end global scope---
